app/furniture/marketplace/split_job.rb

```diff
@@ -0,0 +1,22 @@
+class Marketplace
+  class SplitJob < ApplicationJob
```

Comment:
There is a world where I would have a `Marketplace::SplitPayment` class which does everything but the Sidekiqy-bits; so the database munging can be handled independently of the background jarb-bits.

This encapsulation makes the database / api bits far easier to test... But I didn't do that because we test this through both the `request` and `system` specs covering the `Marketaplace`